lib/fetch/index.js

```diff
@@ -1742,8 +1742,9 @@ function httpNetworkFetch (
             controller.enqueue(new Uint8Array(bytes))
 
             // 8. If stream is errored, then terminate the ongoing fetch.
-            if (stream[kError]) {
-              this.context.terminate({ reason: stream[kError] })
+            // TODO (perf): This check is slow,
+            if (/state: 'errored'/.test(nodeUtil.inspect(stream))) {
+              this.context.terminate()
```

Comment:
Maybe create an issue for this?

Reply:
Where? I think I already created one for @jasnell in node core. But I don't think this is something we can fix unless undici is in core.

Reply:
https://github.com/nodejs/node/pull/41121